tests/unit/lms/services/canvas_api/_authenticated_test.py

```diff
@@ -161,12 +161,13 @@ def test_ok(self, token_method, http_session, token_response):
         token = token_method("code")
         assert token == token_response["access_token"]
 
-    @pytest.mark.parametrize("bad_value", [-1, 0, "string"])
-    def test_bad_expires_in(
-        self, token_method, http_session, token_response, bad_value
+    def test_invalid_access_token_response(
+        self, token_method, http_session, token_response
     ):
+        del token_response["access_token"]
+
         http_session.send.return_value = factories.requests.Response(
-            status_code=200, json_data=dict(token_response, expires_in=bad_value)
+            status_code=200, json_data=dict(token_response)
```

Comment:
This test was indirectly testing some details ot `TokenResponseSchema` (now `OAuthTokenResponseSchema`): the different types of `expires_in` values that are considered invalid. We don't need to test those details here as `OAuthTokenResponseSchema` now has its own unit tests that cover it all. But we _do_ need a test for what `AuthenticatedClient` does  when`OAuthTokenResponseSchema` raises.

Normally we'd test that by configuring the mock `HTTPService` to raise but that's not how the tests in `tests/unit/lms/services/canvas_api/` work: they're integrated tests rather than our usual style of isolated unit tests. The tests call the real `OAuthTokenResponseSchema` indirectly and they don't use `HTTPService` at all, they indirectly call some code in another file that calls the real `requests` and applies the schema to the response.

So I've just renamed this test to `test_invalid_access_token_response()` and made it test the one most obvious kind of invalid response: one with no access token.